index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title> Intuit account</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+        <link rel="stylesheet noopener noreferrer" type="text/css" href="stylesheets/style.css">
+        <link href="/your-path-to-fontawesome/css/all.css" rel="stylesheet noopener noreferrer">
+        <link rel="stylesheet noopener noreferrer" href="path/to/font-awesome/css/font-awesome.min.css">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet noopener noreferrer">
+    </head>
+
+    <body>
+        <header>
+            <section class="sign-in-info">
+                <div class="container">
+                <div class="common-styles">
+                <div class="signin-account">
+                    <p>Already have an Intuit account?</p>
+                </div>
+                
+                <div class="forgot-password">
+                    <p><a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3Da9b8df23-0d05-490a-a527-662b045fd445%26adobe_mc%3DMCMID%253D43778899234622625232808106119344280814%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1600358737%26ivid%3Da9b8df23-0d05-490a-a527-662b045fd445#" rel="noopener noreferrer" target="_blank">I forgot my User ID or Password</a></p>
+                </div>
+            </div>
+            
+            <div class="signin-btn">
+             <button type="button" name="sign-in" value="Sign In"> <i class="material-icons">lock</i> Sign In</button>
+            </div>
+        </div> 
+            </section>
+        </header>
+        
+
+        <section>
+            <div class="logo">
+                <img src="images/descarga1.png" alt="intuit-logo" >
+            </div>
+          <div class="logos">
+              <div class="column">
+                <img src="images/descarga2.png" alt="mint-logo">
+              </div>
+              <div class="column">
+                <img src="images/descarga3.png" alt="quicklink-logo">
+              </div>
+                <div class="column">
+                <img src="images/descarga4.png" alt="turbotax-logo">
+              </div>
+          </div>
+        </section>
+
+        <section class="sign-up">
+                <div class="form-size">
+                    <div class="form-header">
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <h3>One account for everything Intuit, including Mint.</h3>
```

Comment:
[OPTIONAL] Please change the font-weight of `h1` and `h3` tags and also play around with a different font like 'Helvetica'.

[OPTIONAL] You could also use `text-align: center` property to center the elements rather than using approximate margin values.